packages/pangea-node-sdk/tests/integration/vault.test.ts

```diff
@@ -484,16 +480,24 @@ it("RSA encrypting life cycle", async () => {
 });
 
 it("AES encrypting life cycle", async () => {
-  const algorithm = Vault.SymmetricAlgorithm.AES;
+  const algorithms = [
+    Vault.SymmetricAlgorithm.AES128_CBC,
+    Vault.SymmetricAlgorithm.AES256_CBC,
+    Vault.SymmetricAlgorithm.AES128_CFB,
+    Vault.SymmetricAlgorithm.AES256_CFB,
+    Vault.SymmetricAlgorithm.AES256_GCM,
+  ];
   const purpose = Vault.KeyPurpose.ENCRYPTION;
-  try {
-    const id = await symGenerateDefault(algorithm, purpose);
-    await encryptingCycle(id);
-    await vault.delete(id);
-  } catch (e) {
-    console.log(`Failed symmetric encrypting life cycle with ${algorithm} and ${purpose}`);
-    expect(false).toBeTruthy();
-  }
+  algorithms.forEach(async (algorithm) => {
+    try {
+      const id = await symGenerateDefault(algorithm, purpose);
+      await encryptingCycle(id);
+      await vault.delete(id);
+    } catch (e) {
+      console.log(`Failed symmetric encrypting life cycle with ${algorithm} and ${purpose}`);
+      expect(false).toBeTruthy();
+    }
+  });
 });
```

Comment:
Note this is kind of a common gotcha: this isn't actually doing the asynchronous `for` loop you might be expecting. You should do this instead:

```suggestion
  for (const algorithm of algorithms) {
    try {
      const id = await symGenerateDefault(algorithm, purpose);
      await encryptingCycle(id);
      await vault.delete(id);
    } catch (e) {
      console.log(`Failed symmetric encrypting life cycle with ${algorithm} and ${purpose}`);
      expect(false).toBeTruthy();
    }
  }
});
```

Reply:
Updated